source/documentation/guidance/upgrading-from-cflinuxfs2.md

```diff
@@ -0,0 +1,25 @@
+## Upgrading from `cflinuxfs2`
```

Comment:
Would change to "Upgrade from cflinuxfs". Users only do this once so the verb should be imperative rather than a gerund. Also, putting code case into headings looks weird so I would not put backticks around "cflinuxfs2"

Reply:
I agree about the code case. "Upgrade from cflinuxfs" sounds strange because people are still using `cflinuxfs3` afterwards. "Upgrading cflinuxfs"? I'd prefer to stick with the specific "Upgrade from cflinuxfs2" because these instructions assume that is the version being upgrade from.

Reply:
Ah yes I agree my mistake. "Upgrade from cflinuxfs2"

Reply:
Brilliant, I've done that 👍